FitGoal/FitGoal/ExerciseDetailViewController/ExerciseDetailViewController.swift

```diff
@@ -0,0 +1,111 @@
+        self.exerciseCollectionView.backgroundColor = .clear
+        
+        exerciseCollectionView.register(ExercisePreviewCell.self, forCellWithReuseIdentifier: ExercisePreviewCell.identifier)
+        
+        exerciseCollectionView.register(ExerciseDescriptionCell.self, forCellWithReuseIdentifier: ExerciseDescriptionCell.identifier)
+    }
+    
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        exerciseCollectionView.frame = view.bounds
+        startExerciseButton.frame = CGRect (x: 16, y: view.bounds.maxY - 84, width: view.bounds.width - 32, height: 54)
+        startExerciseButton.layer.cornerRadius = startExerciseButton.bounds.height/2
+    }
+}
+
+extension ExerciseDetailViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        guard let exerciseSection = ExerciseDetailSection(rawValue: section) else {
+            fatalError("Section value should have a corresponding case in the HomeSection enum")
+        }
+        
+        switch exerciseSection {
+        case .preview:
+            return 1
+        case .description:
+            return 1
+        }
+    }
+    
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        return 2
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        
+        guard let exerciseSection = ExerciseDetailSection(rawValue: indexPath.section) else {
+            fatalError("Section value should have a corresponding case in the HomeSection enum")
+        }
+        
+        switch exerciseSection {
+        case .preview:
+            guard let cell = exerciseCollectionView.dequeueReusableCell(withReuseIdentifier: ExercisePreviewCell.identifier, for: indexPath) as? ExercisePreviewCell else { fatalError() }
+            cell.exercise = exercise
+            return cell
+        case .description:
+            guard let cell = exerciseCollectionView.dequeueReusableCell(withReuseIdentifier: ExerciseDescriptionCell.identifier, for: indexPath) as? ExerciseDescriptionCell else { fatalError() }
+            cell.exercise = exercise
+            return cell
+        }
+    }
+}
+
+extension ExerciseDetailViewController: UICollectionViewDelegateFlowLayout {
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        guard let exerciseSection = ExerciseDetailSection(rawValue: indexPath.section) else {
+            fatalError("Section value should have a corresponding case in the HomeSection enum")
+        }
+        
+        switch exerciseSection {
+        case .preview:
+            return CGSize(width: view.bounds.width - 32, height: 360)
+        case .description:
+            return CGSize(width: view.bounds.width - 32, height: 360)
```

Comment:
You shouldn't use a static height for the cell displaying the description. What will happen if the description is bigger than the allowed text height?